build.gradle

```diff
@@ -35,6 +35,11 @@ allprojects {
         maven { url 'https://repo.wea-ondara.net/repository/public/' }
         maven { url 'https://repo.extendedclip.com/content/repositories/placeholderapi/' }
         maven { url 'https://jitpack.io' }
+        maven { url 'https://repo.william278.net/releases/' }
+    }
+
+    dependencies {
+        implementation 'net.william278:papiproxybridge:1.2.1'
```

Comment:
this should be compileOnly; we don't want to include the API in the built project. It should also be moved to the `common` build.gradle instead of the master project build.gradle since PAPIProxyBridge classes are only effectively needed there.